docs/content/stories/Untitled.ipynb

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
can you rename the notebook to "TKC_Mag.ipynb" ? or something along those lines?